lib/CPANRatings/Model/SearchCPAN.pm

```diff
@@ -9,130 +9,133 @@ use URI::Escape ();
 my $json = JSON->new->utf8;
 
 sub new {
-  my ($proto, %args) = (shift, @_);
-  my $class = ref $proto || $proto;
-  my $self = { };
-  bless( $self, $class);
+    my ($proto, %args) = (shift, @_);
+    my $class = ref $proto || $proto;
+    my $self  = {};
+    bless($self, $class);
 }
 
 sub search_distribution {
-  shift->_search('dist', @_);
+    shift->_search('dist', @_);
 }
 
 sub _search {
-  my ($self, $mode, $query) = @_;
+    my ($self, $mode, $query) = @_;
```

Comment:
Looks like this introduced some unnecessary whitespace changes.

Reply:
yeah, it was whatever my editor did. I tried doing perl tidy on the files but that just made different changes... 🤷